addons/hr/hr.py

```diff
@@ -247,10 +247,11 @@ def onchange_department_id(self, cr, uid, ids, department_id, context=None):
         return {'value': value}
 
     def onchange_user(self, cr, uid, ids, user_id, context=None):
-        work_email = False
         if user_id:
-            work_email = self.pool.get('res.users').browse(cr, uid, user_id, context=context).email
-        return {'value': {'work_email': work_email}}
+            user = self.pool['res.users'].browse(cr, uid, user_id, context=context)
+            return {'value': {'name': user.name, 'work_email': user.email, 'image_medium': user.image_medium}}
```

Comment:
"As specified by @tde-banana-odoo , you must set user.image"
Oh you probably didn't see my comment after I closed previous PR. image doesn't work. I did test,  only image_medium, works, that's why i left it unchanged.

As per split lines, sorry I didn't get that odony mean indentation.